benchmark/main.cpp

```diff
@@ -1015,20 +999,19 @@ static int benchmark( int argc, char *argv[], mbedtls_platform_context* ctx )
 }
 
 int main(void) {
-    mbedtls_platform_context platform_ctx;
     int exit_code = MBEDTLS_EXIT_FAILURE;
 
-    if((exit_code = mbedtls_platform_setup(&platform_ctx)) != 0) {
+    if((exit_code = mbedtls_platform_setup(NULL)) != 0) {
```

Comment:
Again, I think you should remove the assignment to `exit_code`, and initialise it to `MBEDTLS_EXIT_SUCCESS` instead.